gravitee-apim-portal-webui/src/app/pages/application/application-creation/application-creation-step3/application-creation-step3.component.ts

```diff
@@ -118,6 +118,17 @@ export class ApplicationCreationStep3Component implements OnInit, OnDestroy {
         this.planForm.get('general_conditions_accepted').clearValidators();
       }
 
+      if (this.selectedPlan?.mode.toUpperCase() === 'PUSH') {
+        this.planForm.get('entrypoint').setValidators(Validators.required);
+      } else {
+        this.planForm.get('channel').setValue(null);
+        this.planForm.get('channel').clearValidators();
+        this.planForm.get('entrypoint').setValue(null);
+        this.planForm.get('entrypoint').clearValidators();
+        this.planForm.get('entrypointConfiguration').clearValidators();
+        this.planForm.get('entrypointConfiguration').setValue(null);
```

Comment:
After `clearValidators()`, `this.planForm.updateValueandValidity()` must be called for the new validators to take effect.